app/routes.py

```diff
@@ -618,6 +634,19 @@ def view_harvest_sources():
 @mod.route("/harvest_source/config/edit/<source_id>", methods=["GET", "POST"])
 @login_required
 def edit_harvest_source(source_id: str):
+    if request.is_json:
+        source = db.get_harvest_source(source_id)
```

Comment:
I don't think we need this check 638-640 b/c the interface does it already

Reply:
removed.